sdk/core/azure-core-tracing-opencensus/src/samples/AsyncListKeyVaultSecrets.md

```diff
@@ -12,7 +12,7 @@ Sample uses **[opencensus-impl][opencensus_impl]** as implementation package and
 <dependency>
     <groupId>com.azure</groupId>
     <artifactId>azure-security-keyvault-secrets</artifactId>
-    <version>4.0.0</version>
+    <version>4.1.0-beta.1</version>
```

Comment:
Overall, get in touch with the SDK owners to make sure these versions are updated correctly, we shouldn't link these to invalid package versions.

Reply:
@g2vinay could you confirm the version here for the secrets library.

Reply:
it will be 4.0.1 most likely.
Will update it here when we update secrets version before releasing.